install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -605,7 +605,7 @@ data:
   enable-l2-neigh-discovery: {{ .Values.l2NeighDiscovery.enabled | quote }}
 {{- end }}
 {{- if hasKey .Values.l2NeighDiscovery "refreshPeriod" }}
-  arping-refresh-period: {{ .Values.l2NeighDiscovery.refreshPeriod | quote }}
+  refreshPeriod: {{ .Values.l2NeighDiscovery.refreshPeriod | quote }}
```

Comment:
we can keep the attribute name as arping-refresh-period to avoid breaking change (mainly to backport this PR to v1.11).

Reply:
OK